src/Http/Productions/ProductionsWriteController.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace CultuurNet\UDB3\Http\Productions;
+
+use Broadway\CommandHandling\CommandBusInterface;
+use CultuurNet\UDB3\Event\Productions\GroupEventsAsProduction;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+class ProductionsWriteController
+{
+    /**
+     * @var CommandBusInterface
+     */
+    private $commandBus;
+
+    public function __construct(CommandBusInterface $commandBus)
+    {
+        $this->commandBus = $commandBus;
+    }
+
+    public function create(Request $request): Response
+    {
+        $data = json_decode($request->getContent(), true);
+
+        $command = new GroupEventsAsProduction(
+            $data['eventIds'],
+            $data['name']
```

Comment:
We should add some validation for this deserialization of JSON.

Here's a good example of how we do that when creating a new organizer:
- A class for the payload as an object: https://github.com/cultuurnet/udb3-silex/blob/master/src/Http/Deserializer/Organizer/OrganizerCreationPayload.php
- A data validator class: https://github.com/cultuurnet/udb3-silex/blob/master/src/Http/Deserializer/Organizer/OrganizerCreationPayloadDataValidator.php (This is something custom right now. We should start using something like respect instead in the future, but let's maybe stick to this approach for now so as not to widen the scope of this ticket too much and to maintain some consistency)
- A deserializer to convert the json to the payload class: https://github.com/cultuurnet/udb3-silex/blob/master/src/Http/Deserializer/Organizer/OrganizerCreationPayloadJSONDeserializer.php (which uses the data validator internally)

Additionally, this JSON payload is basically the JSON representation of the command itself. So what you could do is _not_ have a class for the payload, and just have the deserializer return the command instead.

If you do that, you could either dispatch the command here manually if you like, or use https://github.com/cultuurnet/udb3-silex/blob/master/src/Http/CommandDeserializerController.php instead. Here's an example how we use those for the exports: https://github.com/cultuurnet/udb3-silex/blob/master/app/Export/ExportControllerProvider.php#L14

Although I'm also a fan of having it a bit more explicit, so having a `ProductionsWriteController::create()` which deserializes the JSON to a command and dispatches it instead of re-using `CommandDeserializerController` is also totally okay for me. Maybe check how often we actually use `CommandDeserializerController` and if it's very little, do keep it explicit like you do right now.

Even having a deserializer is no strict requirement for me in this scenario. It's nice for payloads that share a lot of complex properties like calendar info etc but in this case, it's a pretty straightforward 1-1 conversion.

Data validation is the only must-have I think, which you can also achieve by injecting the validator directly.